cli/connhelper/ssh/ssh.go

```diff
@@ -23,30 +23,30 @@ func parseSSHURL(daemonURL string) (*sshSpec, error) {
 		return nil, err
 	}
 	if u.Scheme != "ssh" {
-		return nil, errors.Errorf("expected scheme ssh, got %s", u.Scheme)
+		return nil, errors.Errorf("ssh connhelper expected scheme ssh, got %s", u.Scheme)
```

Comment:
Perhaps use `%q` instead of `%s` so that the actual value is quotes (clearer to see what the problematic value is)

Reply:
done